pi-blaster.c

```diff
@@ -18,6 +18,16 @@
  * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 */
 
+#define _POSIX_C_SOURCE 200809L
+#define _XOPEN_SOURCE   700
+#define _BSD_SOURCE
+
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#else
+static char VERSION[] = "0.1.1";
```

Comment:
We are probably going to forget to update this. Maybe we should just replace it with:

````
static char VERSION[] = "SNAPSHOT";
````